.github/workflows/Documentation.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - 'master'
     tags: '*'
+  workflow_dispatch:
```

Comment:
@cpfiffer This should allow us to rebuild the docs manually whenever needed (see https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)

Reply:
Excellent!